crates/turborepo-signals/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "turborepo-signals"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+futures = "0.3.30"
```

Comment:
Any particular reason we're using a different version for futures instead of the workspace root version?

Reply:
I was matching the version used in [`turborepo-lib`](https://github.com/vercel/turborepo/pull/9995/files#diff-217f728c3ab310388e85c7ed8a02edc124d9b28c82b9cf623a41b94204848d41L61). I can update both to use workspace version in a followup PR.